app/views/partials/inputDropdown2.scala.html

```diff
@@ -0,0 +1,46 @@
+@* dropdownOptions as Tuple until data model clearer *@
```

Comment:
Why do we need another inputDropdown form. Can you not change the original and update the other pages that are using it. Or if not can you give it a name which describes what it's used for.

Reply:
Actually, I thought to remove the old one later, but now, I have updated the old pages to use the new dropdown and kept the same name, thanks